### DISCUSSION TITLE
Content adjustments for the first release

### DISCUSSION THREAD

**Discussion by fcoveram** [WordPress/openverse#36] [Ideas]
While reviewing the internal pages, content not related to a search but referring to Openverse as a tool, I discovered some content inconsistencies that we can improve. Here are my thoughts and ideas.

## Showing the same on different pages
On the **Our Story** page, we explain Openverse’s purpose, how it started, and sources connected. I like this approach, but the Sources table is also visible on **Sources** page with more details. 

💡 Idea: Since **Sources** page is focused on the integration procedure and a call for adding new ones, I suggest removing the source table on **Our Story** but keeping the table on Sources.

<img width="1487" alt="Source table" src="https://user-images.githubusercontent.com/895819/123849382-bbd04b80-d8e6-11eb-9e31-ded3f3aff3af.png">

## Make more visible the feedback call
The feedback form is a Resources item on the menu. We can make the call more evident and the chances of user responses might increase.

💡 Idea: Move the **Feedback** page out of Resources, in the same line as About, Resources, and Extension.

<img width="349" alt="Feedback item" src="https://user-images.githubusercontent.com/895819/123849465-d276a280-d8e6-11eb-9288-77d7af778d17.png">

## Being more specific about Extension
The extension link has a label that is not fully clear about what users will see. Extension of what? What might features be associated with this extension?

💡 Idea: Rename as Browser extension since are browser-only features.

<img width="239" alt="Extension external link" src="https://user-images.githubusercontent.com/895819/123849525-e4584580-d8e6-11eb-8fde-b128b9eef17a.png">

❓ Question: Should we move the Browser extension landing page to some Openverse domain? I envision that in the mid-term lapse, other extensions might come.

**Comment by zackkrida** [comment]
Feedback on specific action items:

## I agree with

- Removing the source table on the Our Story page
- Rename as 'Browser extension' since are browser-only features.
  - I'd prefer 'web extension' since it is shorter
- Move the Browser extension landing page to some Openverse domain
	- I'd be fine with making this a page on wordpress.org/openverse instead of an external site. I don't think it needs it's own development environment and build process and all of that.

## I'd like to talk about

> Move the Feedback page out of Resources, in the same line as About, Resources, and Extension.

I'd prefer to leave it where it is, but make it more visible in places where users might want to provide feedback. Some ideas:

- Bottom of search results (somewhat related to https://github.com/WordPress/openverse-frontend/issues/85)
- On single result pages

Alternatively, I'd be fine with a 'feedback' global tab	on the bottom left of the page, like a floating thing, if we want to collect a lot of feedback.

**Reply by fcoveram** [reply]
> I'd prefer 'web extension' since it is shorter

Great. I like it.

> I'd be fine with making this a page on wordpress.org/openverse instead of an external site. I don't think it needs its own development environment and build process and all of that.

Also agree. Let's make an internal page then.

## Asking for feedback
I like both ideas. I will consider them for the audio integration mockups.

Regarding the float element, I'm not sure about it since most static-across-site elements tend to disappear to users' sight after using the site. It has more impact presenting the call during the search journey, like the first idea you pointed.

**Reply by zackkrida** [reply]
> Regarding the float element, I'm not sure about it

Great, because I don't love that idea haha 😄

**Comment by krysal** [comment]
I also like all the ideas but just not sure about placing the Feedback page at the same level of About, Resources... Placing links to it on results and single result pages sounds like a good move.

**Reply by fcoveram** [reply]
Agree. I also think that the feedback call should be placed within the user search instead of the navigational element. Plus, it is not a Resource, then it makes more sense the two ideas made by @zackkrida
